CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## 51 Release Notes
 
-### Bugfixes
+### Bug fixes
```

Comment:
I made this change while resolving the changelog merge conflict, so blame me if you hate it

Reply:
I am not a native speaker so any corrections are more than welcome!